scripts/try-copy-license.cmake

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2015 Aaditya Kalsi
+# All rights reserved.
+
```

Comment:
Please add `cmake_minimum_required(VERSION 3.0)`. I've found recently that this may make script behaves differently.